### ISSUE TITLE
Any way of using as a presence detection for wifi devices on the network?

### ISSUE THREAD

**Issue by thefunkygibbon** [Nekmo/amazon-dash#174]
Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with amazon-dash)
- [x ] Feature request (request for a new functionality)
- [x] Question
- [ ] Other

So i was thinking, since this is just sniffing the network for the addition of specific mac addresses appearing and then informing Home Assistant (and others). I was wondering if it would be possible to do the same to alert HA that the person has Joined the network and then notify if they leave the network.  Effectively becoming a better network presence sensor than using NMAP/ICMP

**Comment by Nekmo** [comment]
Yes, It can be used as a presence sensor :) However, if the device sends reconnection packets, the function could be executed several times.

**Comment by chrissv** [comment]
How would this work in practice? Wouldn't you have to keep pressing the Dash button to announce your "presence" ?
